ui/tests/acceptance/enterprise-kmse-test.js

```diff
@@ -0,0 +1,45 @@
+import { module, test } from 'qunit';
+import { setupApplicationTest } from 'ember-qunit';
+import { click, fillIn } from '@ember/test-helpers';
+import authPage from 'vault/tests/pages/auth';
+import logout from 'vault/tests/pages/logout';
+import mountSecrets from 'vault/tests/pages/settings/mount-secret-backend';
+import { setupMirage } from 'ember-cli-mirage/test-support';
+
+module('Acceptance | Enterprise | kmse', function (hooks) {
```

Comment:
Thanks for adding this acceptance test! One nit: our other tests related to this secrets engine call it `keymgmt`, can we keep the naming consistent with this one?

Reply:
Sure thing I updated it.